modules/dqt/jsx/components/stepper.js

```diff
@@ -0,0 +1,200 @@
+import React from 'react';
```

Comment:
@HenriRabalais do we have any similar components in LORIS already?

Reply:
@driusan I'm pretty sure there isn't anything like this in Loris. I think I asked @HenriRabalais while also showing him the stepper I made.